.github/workflows/dump-config.yml

```diff
@@ -3,8 +3,22 @@ name: Dump config
 on: [push, pull_request]
 
 jobs:
+  check-token:
+    name: Check for token
+    runs-on: ubuntu-latest
+    outputs:
+      token-held: ${{ steps.set_output.outputs.token_held }}
+    steps:
+      - name: check for token
+        id: set_output
+        run: '[ -z "$TOKEN_HELD" ] || echo "token_held=true" >> "$GITHUB_OUTPUT"'
+        env:
+          TOKEN_HELD: ${{ secrets.SHOPIFY_GH_ACCESS_TOKEN }}
```

Comment:
I tried a secret that doesn't exist, and [successfully skipped dumping the config](https://github.com/Shopify/ruby-style-guide/actions/runs/4589514822).

```yaml
TOKEN_HELD: ${{ secrets.EXAMPLE }}
```